frontend/packages/component-library/src/actionBlock/ActionBlock.tsx

```diff
@@ -0,0 +1,160 @@
+import classNames from 'classnames';
+import {HTMLAttributes} from 'react';
+
+import {LinkButton, LinkButtonProps} from '@/button';
+import {
+  Heading3,
+  BodyThreeText,
+  OverlineTwoText,
+  StrongText,
+} from '@/typography';
+
+import moduleStyles from './actionBlock.module.scss';
+
+export interface ActionBlockProps extends HTMLAttributes<HTMLDivElement> {
+  /** Action Block title */
+  title?: string;
+  /** Action Block description */
+  description?: string;
+  /** Action Block image */
+  image?: string;
+  /** Action Block overline */
+  overline?: string;
+  /** Action Block Details */
+  details?: {
+    /** Detail label */
+    label: string;
+    /** Detail text */
+    description: string;
+  };
+  /** Primary button props */
+  primaryButton?: LinkButtonProps;
+  /** Secondary button props */
+  secondaryButton?: LinkButtonProps;
+  /** Action Block background */
+  background?: 'primary' | 'secondary';
+  /** Action Block custom className */
+  className?: string;
+}
+
+export const getImage = (image?: string) => {
+  if (!image) return null;
+  return (
+    <figure>
+      <img src={image} alt="" loading={'lazy'} />
+    </figure>
+  );
+};
+
+export const getDetail = (details?: {label: string; description: string}) => {
+  if (!details) return null;
+  return (
+    <BodyThreeText className={classNames(moduleStyles.detail)}>
+      <StrongText>{`${details.label}: `}</StrongText>
+      {details.description}
+    </BodyThreeText>
+  );
+};
+
+export const getButtons = (
+  primaryButton?: LinkButtonProps,
+  secondaryButton?: LinkButtonProps,
+) => {
+  if (!primaryButton) return null;
+  return (
+    <div className={moduleStyles.buttonWrapper}>
+      {primaryButton && (
+        <LinkButton
+          {...primaryButton}
+          type="primary"
+          size="m"
+          color="purple"
+          text={primaryButton.text}
+          href={primaryButton.href}
+          ariaLabel={primaryButton.ariaLabel}
+        />
+      )}
+      {secondaryButton && (
+        <LinkButton
+          {...secondaryButton}
+          type="secondary"
+          size="m"
+          color="black"
+          text={secondaryButton.text}
+          href={secondaryButton.href}
+          ariaLabel={secondaryButton.ariaLabel}
```

Comment:
nit: please move `{...secondaryButton}` to the end of props declaration. This will allow to overwrite default props listed here if needed
```suggestion
          type="secondary"
          size="m"
          color="black"
          text={secondaryButton.text}
          href={secondaryButton.href}
          ariaLabel={secondaryButton.ariaLabel}
          {...secondaryButton}
```